service.js

```diff
@@ -0,0 +1,50 @@
+sqlite3 = require("sqlite3").verbose();
+
+class DatabaseService {
+  _db;
+  constructor() {
+    this._db = new sqlite3.Database(
+      "./linkBank.db",
+      sqlite3.OPEN_READWRITE,
+      (err) => {
+        if (err) return console.error(err.message);
+        console.log("DB connection succesful");
+      }
+    ); //connects to the database
+
+    //this._db.run("CREATE TABLE oLinks(id, originLink)");//created my table
+  }
+  intoDB(id, link) {
```

Comment:
method names should be more semantic (carry a meaning). This might be called, e.g. insertLink.

With this strategy you should be able to read your code like english and have 0 inline comments (that's what we aim for)